README.md

```diff
@@ -12,6 +12,62 @@ If there is anything you are looking for that is not covered here you should ref
 
 ----
 
+### File structure ###
+
+You should not define multiple types (ie class, struct, enum...) in the same file; each type should have its own file.
```

Comment:
Not sure enums get into that list. Also what is the '...' for? Also if you were to use '...' should use ie -> eg. If the list is finite then 'ie' is fine. Makes sense? If not let's chat